blog/_posts/2021-01-15-snap-announcement.md

```diff
@@ -0,0 +1,158 @@
+
+A track in Snapcraft terms is a a separate branch of a project that can be downloaded instead of the default release.
+The default release is called "latest", and it's stable channel will generally follow the newest stable ROOT release.
+As a result, users will automatically update to newer branches of ROOT.
+However, in some scenario's people may like to use an older release, and tracks could be used to provide this in the future.
+
+If there's a demand to produce these tracks, please provide some feedback and it can be looked into!
+The following example syntax would be usable if/when tracks are declared.
+
+```
+sudo snap install root-framework --channel=v6-22/stable
+```
+
+### Multiple Installations of the ROOT Snap
+[https://snapcraft.io/docs/parallel-installs](https://snapcraft.io/docs/parallel-installs) 
+
+[https://snapcraft.io/docs/commands-and-aliases](https://snapcraft.io/docs/commands-and-aliases)
+
+This feature is still experimental, but it's possible to have both the ROOT stable snap and the ROOT nightly snap alongside each other.
+However to make proper use of this functionality, it helps to understand snap aliases.
+
+The `root` command itself is an alias, because the snap package is called `root-framework`. When installed from the Snap Store, an alias is created automatically between `root` and `root-framework`.
+For the other binaries such as `hadd`, the original names are namespaced, so hadd's original name is `root-framework.hadd`.
+When you install extra instances of a snap, you must decide which aliases you wish to use manually. 
+
+
+```
+sudo snap set system experimental.parallel-instances=true
+sudo snap install root-framework
+sudo snap install root-framework_nightly --edge --unaliased
+```
+
+Whilst `root` will point to the stable version, you can run `sudo snap prefer root-framework_nightly` so that the next invocation of `root` will be from the nightly branch.
+You can also alias individual commands, or simply use the unaliased binary names, such as `root-framework_nightly.root`.
+
+It's preferable to use snap aliases rather than Bash aliases because they work by placing binaries in the $PATH, for all system users.
+
+The additional installations of snaps will all have their own unique $HOME values, so can have differing `rootlogin.c` files for example.
+
+### Windows Subsystem for Linux 2.
+[https://discourse.ubuntu.com/t/using-snapd-in-wsl2/12113](https://discourse.ubuntu.com/t/using-snapd-in-wsl2/12113)
+
+Officially, running snaps on WSL2 is unsupported.
+This is because WSL2 has a custom init system rather than systemd.
+Unofficially, it's possible to get it working reasonably well anyway, but this is not directly supported.
+
+Having personally tried it, it's possible to get JupyROOT in the snap running in a web browser on Windows, and for some people, the snap on WSL2 might make sense.
+
+Other virtual machine platforms, such as Virtualbox should be able to install Snaps fine.
```

Comment:
fine -> without issues (less colloquial)

Reply:
I've made this change